### DISCUSSION TITLE
No such file or directory: 'train.spacy'

### DISCUSSION THREAD

**Discussion by NaziaToma** [explosion/spaCy#9040] [Help: Coding & Implementations] [answered]
```
py -m spacy init fill-config base_config.cfg config.cfg
py -m spacy train config.cfg --output ./output --paths.train ./train.spacy --paths.dev ./dev.spacy
```

After running these two commands I am getting -

    FileNotFoundError: [Errno 2] No such file or directory: 'train.spacy'

How should I resolve this? Also, If I want to add data augmentation then after installing this config file, should my code look like this for training? I am using VS Code.

```
from spacy.training.example import Example
for iter in range(100):
    random.shuffle(training_data["annotations"])
    for text, annotations in training_data["annotations"]:
        # create Example
            doc = nlp.make_doc(text)
            example = Example.from_dict(doc, annotations)
        # Update the model
            nlp.update([example], drop=0.3)
```

**Comment by polm** [comment, marked as answer]
Does the `train.spacy` file exist? Did you create training data?

If you have generated a config file, you should be able to use `spacy train`, and you do not need to write a training loop at all. In fact we recommend you do not write training loop and use `spacy train` instead. I am not sure what your training loop has to do with augmentation, but maybe look at the [augmentation section](https://spacy.io/usage/training#data-augmentation) of the docs - it has examples using a config, and also custom code.

**Reply by NaziaToma** [reply]
I have annotated training data that I used before. How do I get or use the train.spacy file?

**Reply by polm** [reply]
Please see the recently added [training data section](https://spacy.io/usage/training#training-data) in the docs. Basically you need to save a DocBin, which is a collection of Docs.
